packages/twenty-server/src/modules/workflow/workflow-builder/workflow-builder.workspace-service.ts

```diff
@@ -122,6 +124,33 @@ export class WorkflowBuilderWorkspaceService {
     );
   }
 
+  private async computeRecordCrudOutputSchema<Entity>({
+    objectType,
+    operationType,
+    workspaceId,
+    objectMetadataRepository,
+  }: {
+    objectType: string;
+    operationType: string;
+    workspaceId: string;
+    objectMetadataRepository: Repository<ObjectMetadataEntity>;
+  }) {
+    const recordOutputSchema = await this.computeRecordOutputSchema<Entity>({
+      objectType,
+      workspaceId,
+      objectMetadataRepository,
+    });
+
+    if (operationType === WorkflowRecordCRUDType.FIND) {
+      return {
+        first: recordOutputSchema,
+        last: recordOutputSchema,
```

Comment:
totalCount missing but it appears on figma ->
![image](https://github.com/user-attachments/assets/fa81d458-5b17-48cb-8c03-7f2f716271c9)